planning/behavior_path_planner/src/scene_module/pull_over/pull_over_module.cpp

```diff
@@ -699,6 +714,18 @@ BehaviorModuleOutput PullOverModule::planWaitingApproval()
 
   const auto distance_to_path_change = calcDistanceToPathChange();
   updateRTCStatus(distance_to_path_change.first, distance_to_path_change.second);
+
+  uint16_t direction;
+  if (out.turn_signal_info.turn_signal.command == TurnIndicatorsCommand::ENABLE_LEFT) {
+    direction = SteeringFactor::LEFT;
+  } else {
+    direction = SteeringFactor::RIGHT;
```

Comment:
When there is no turn signal, it is RIGHT, is it OK?

![Screenshot from 2022-09-13 15-22-23](https://user-images.githubusercontent.com/39142679/189826614-c66c2eda-7ec8-4210-a8b9-1ac074401cc9.png)

Reply:
`out.turn_signal_info.turn_signal.command` is not set in this function, so always be `RIGHT`.

Reply:
Pull_over can be currently only done to the left shoulder, so it may be ok to always use `LEFT`.

Reply:
@kosuke55 looks like pull out and pull over also have to disable turn indicator, I will change the code

Reply:
I add STRAIGHT in this commit [da2db2d](https://github.com/autowarefoundation/autoware.universe/pull/1706/commits/da2db2d0f9050bd975ad170a0e7f98747b17fb97)

Reply:
thaks! please also update docs related to the msgs

Reply:
@kosuke55 you mean in the autoware document?
https://github.com/autowarefoundation/autoware-documentation/pull/154
I putted a changes request in the PR

Reply:
yes, nice!

![image](https://user-images.githubusercontent.com/39142679/189859296-b82dc68f-5be1-40f7-864f-06d558eafe6b.png)